--- razorqt-0.5.2/razorqt-lightdm-greeter/CMakeLists.txt.orig	2013-08-06 13:51:08.903170536 +0200
+++ razorqt-0.5.2/razorqt-lightdm-greeter/CMakeLists.txt	2013-08-06 12:44:49.437998809 +0200
@@ -24,7 +24,7 @@
 # check for new version
 include(FindPkgConfig)
 message(STATUS "Checking for lightdm-qt...")
-pkg_check_modules(LIGHTDM_QT liblightdm-qt-2)
+pkg_check_modules(LIGHTDM_QT liblightdm-qt-3)
 # or lightdm v1 if the version 2 is not found
 if (NOT LIGHTDM_QT_FOUND)
     message(STATUS "lightdm2 for Qt is not found. Trying to find v1")
